### DISCUSSION TITLE
How to convert RTSP to WebRTC or another format supported by browsers?

### DISCUSSION THREAD

**Discussion by VacantHusky** [bluenviron/gortsplib#429] [Questions]
I would like to use a browser to watch rtsp.

**Comment by aler9** [comment]
Hello, [MediaMTX](https://github.com/bluenviron/mediamtx) allows automatic conversion of RTSP streams to WebRTC or HLS. Instructions are here:

https://github.com/bluenviron/mediamtx#web-browsers-1
